_episodes/17-preparation.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Preparing to Teach"
+block: "Building Teaching Skill"
+teaching: 20
+exercises: 25
+questions:
+- "How should I prepare to teach?"
+objectives:
+- "Create a profile for a learner in your workshop."
+- "Critically analyze a learning objective for your workshop."
+- "Identify checkpoints in a lesson for formative assessment."
+keypoints:
+- "To teach effectively, you have to know *who* you are teaching."
+- "Good learning objectives identify specific events that can be evaluated through formative assessment."
+- "A good exercise informs Learners and Instructors when an objective is achieved."
+---
+# Building Teaching Skill
+At this point in the workshop, we have discussed many cognitive principles and teaching practices that guide collaborative 
+development and maintenance of The Carpentries curricula. We hope you are also feeling excited and optimistic about putting 
+those concepts to work in your teaching practice! 
+One thing we have emphasized is that **teaching is a skill** - and a complex one at that. 
+Whether you are new to teaching or skilled in certain methods, adding new features takes takes effort and practice! In this episode, 
+we will walk through some techniques for preparing to teach a workshop that we hope will help you to develop goals
+and implementation plans that are specific to a workshop. 
+
+As with other sections of this training, we will not be discussing technical preparation. Carefully reviewing the content of your workshop is important, and it
+can be useful to anticipate learner questions and update your knowledge accordingly. However, it is common for new Instructors to
+over-prepare on technical content -- which can be endless! -- and under-prepare the learner-centered elements of their teaching practice. If you encounter
+technical questions that you do not have answers to, showing learners how you Google things may earn you some surprisingly positive feedback. On the other hand, neglecting
+to attend to your audience can have a significant negative impact on your workshop.
+
+When you prepare to teach a workshop, 
+we suggest setting aside time *before* deeply reviewing your technical content to think through the learner experience, how
+your teaching will translate into their learning, and how both of you will know when that happens. In this episode, we will provide
+some structure for that preparation.
+
+![A tree diagram of Carpentries instruction and audience in which Instructor Trainers teach Instructors and Instructors teach Learners](../fig/instructor-training-program.png)
```

Comment:
This image might be more helpful in the previous episode about how the carpentries works...